scripts/linters/js_ts_linter_test.py

```diff
@@ -576,6 +578,64 @@ def mock_compile_all_ts_files():
             'allowed to create http requests from files that are not '
             'backend api services.'], self.linter_stdout)
 
+    def test_oppia_root_component(self):
+        def mock_compile_all_ts_files():
+            cmd = (
+                './node_modules/typescript/bin/tsc -outDir %s -allowJS %s '
+                '-lib %s -noImplicitUseStrict %s -skipLibCheck '
+                '%s -target %s -typeRoots %s %s typings/*') % (
+                    js_ts_linter.COMPILED_TYPESCRIPT_TMP_PATH +
+                    'scripts/linters/test_files/', 'true', 'es2017,dom', 'true',
+                    'true', 'es5', './node_modules/@types',
+                    VALID_UNLISTED_SERVICE_PATH)
+            subprocess.call(cmd, shell=True, stdout=subprocess.PIPE)
+
+        compile_all_ts_files_swap = self.swap(
+            js_ts_linter, 'compile_all_ts_files', mock_compile_all_ts_files)
+        with self.print_swap, compile_all_ts_files_swap:
+            js_ts_linter.JsTsLintChecksManager(
+                [], [VALID_UNLISTED_SERVICE_PATH], FILE_CACHE,
+                True).perform_all_lint_checks()
+
+        shutil.rmtree(
+            js_ts_linter.COMPILED_TYPESCRIPT_TMP_PATH, ignore_errors=True)
+
+        self.assert_same_list_elements([
+            'Please import UnlistedService to Oppia Angular Root Component in '
+            './core/templates/components/oppia-angular-root.component.ts',
+        ], self.linter_stdout)
+        self.assert_same_list_elements([
+            'Please add a static class member unlistedService to Oppia Angular'
+            ' Root Component in ',
+        ], self.linter_stdout)
+        self.assert_same_list_elements([
+            'static unlistedService: UnlistedService;',
+        ], self.linter_stdout)
```

Comment:
You can combine these two by adding a comma in between
```suggestion
            'Please add a static class member unlistedService to Oppia Angular'
            ' Root Component in ', 'static unlistedService: UnlistedService;'
        ], self.linter_stdout)
```

Reply:
That didn't work, remember?

Reply:
These lines are combinable, try one more time.